scripts/python/plot_torus_3d.py

```diff
@@ -31,14 +31,11 @@
 import phoebus_utils
 from phoedf import phoedf
 
-def plot_frame(ifname, fname, savefig, geomfile=None, rlim=40, coords='cartesian', draw_bh=True):
-  print(fname)
+def plot_frame_from_phoedf(dfile, ifname, savefig, geomfile=None, rlim=40, coords='cartesian', draw_bh=True, custom_name=None):
```

Comment:
this is long. Can you split it across multiple lines.

Reply:
Done (I used `black`; maybe we should add that to `scripts/bash/format.sh` if `black` exists on the system?)

Reply:
that's a good idea.